workers/unity/Packages/com.improbable.gdk.buildsystem/WorkerBuilder.cs

```diff
@@ -44,9 +47,33 @@ public static void Build()
                         buildEnvironment = BuildEnvironment.Local;
                         break;
                     default:
-                        throw new BuildFailedException("Unknown build target value: " + buildTargetArg);
+                        throw new BuildFailedException("Unknown build environment value: " + buildEnvironmentArg);
                 }
 
+                var buildTargets = buildTargetsArg
+                    .Split(',')
+                    .SkipWhile(target => target == "all")
```

Comment:
If you have something like `all,ios,all` this will throw the unknown build target value. We should enforce that its `all` or one or more specific targets, not a mix (with a more descriptive error message)